config/projects.yml

```diff
@@ -259,7 +259,7 @@ media-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0']
       target_php: 5.6
       versions:
-        symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
+        symfony: ['2.8', '3.0', '3.1']
         doctrine_odm: ['1']
         sonata_core: ['3']
         sonata_admin: ['3']
```

Comment:
when you build media-bundle 3.x you are using a tag of sonata-admin like 3.13.0. This allows to test against sonata-admin 3.x